UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -22,7 +22,7 @@
   SUPPORTED_ARCHITECTURES             = IA32|X64

```

Comment:
I think it may also need to change to IA32|X64|RISCV64

Reply:
It will come up in follow up patches.